README.md

```diff
@@ -20,3 +20,13 @@ cargo sql migrate add <name_of_migration>
 ### Authenticated Requests
 
 The endpoint for retrieving subscribers is locked behind an admin token that is set through the env. Set the env variable `ADMIN_TOKEN` to whatever value you want and just ensure you pass it whenever you are accessing the endpoint inside of the `Authorization` header, using `Bearer <ADMIN_TOKEN>`.
+
+### Configurable Redirect Location
+
+If you would like to configure where `minimail` should send users upon a successful save, the address can be filled in the configuration file by adding something like the following.
```

Comment:
Maybe add a note about origin being the default?

Reply:
Good idea. I’ll add that